docs/source/notebook.rst

```diff
@@ -124,10 +127,13 @@ from the notebook directory or to create a new notebook.
 Creating a new notebook document
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-A new notebook may be created at any time, either from the dashboard, or using
-the `File | New` menu option from within an active notebook. The new notebook
-is created within the same directory and will open in a new browser tab. It
-will also be reflected as a new entry in the notebook list on the dashboard.
+You can create a new notebook at any time, either from the dashboard (as detailed
```

Comment:
Suggested: Combine the beginning of the existing paragraph with the PR recommendations end of the paragraph.

{Existing text} A new notebook may be created at any time, either from the dashboard, or using the File | New menu option from within an active notebook. {PR text}Select the programming language to use in the notebook from a submenu.{End PR text} {PR text}The new notebook is created within the same directory and will open in a new browser tab. This will also create a new entry in the notebook list in the dashboard. {End PR text}